components/text-utility/text-utility.css

```diff
@@ -0,0 +1,52 @@
+@import url("/index.css");
+@import url("/components/sidebar.css");
+
+body {
+    background-image: none;
+}
+.component-container {
+    margin-left: 5rem;
+}
+.component-description {
+    margin-top: 1rem;
+    margin-bottom: 1rem;
+    font-size: 1.5rem;
+}
+.text-utility-container{
+    width: 100%;
+    border: 1px solid var(--border-color);
+    box-shadow: 2px 2px var(--alert-secondary-color);
+    padding: 1rem;
+    margin-bottom: 1rem;
+}
+.text-align p{
+    font-size: 1.5rem;
+ }
+ .text-align{
+     height: 100%;
+     width: 100%;
+ }
+ .centre-align{
```

Comment:
Sugesstion: write class names descriptively.
for ex follow any naming convention, and if not you can always try writing class names as 
attribute-property for ex. align-text is more descriptive than text-align.

Reply:
Sure. I'll refactor the classname.